CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [development]
+
+### Added
+- Support for `on`/`off` replacing `addEventHandler`/`removeEventHandler` in player Version 8
```

Comment:
It is player 7.7 (or 7.8) I guess which introduces on/off? There will be a separate branch and UI 3.0 for player v8 so it is confusing to mention it here

Reply:
Also `version` in lowercase, its not a proper noun.